database/migration/migrateData.ts

```diff
@@ -643,6 +643,229 @@ const migrateData = async () => {
+        'ERROR'
+      );
+
+      ALTER TABLE trigger_queue
+        ALTER COLUMN trigger_type TYPE public.trigger
+        USING trigger_type::text::public.trigger;    
+        
+      ALTER TABLE template_action
+        ALTER COLUMN trigger TYPE public.trigger
+        USING trigger::text::public.trigger;
+
+      DROP TRIGGER IF EXISTS application_trigger ON application;
+
+      ALTER TABLE application
+          ALTER COLUMN TRIGGER TYPE public.trigger
+          USING TRIGGER::text::public.trigger;
+      
+      CREATE TRIGGER application_trigger
+          AFTER INSERT OR UPDATE OF trigger ON public.application
+          FOR EACH ROW
+          WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
+          EXECUTE FUNCTION public.add_event_to_trigger_queue ();
+      
+      DROP TRIGGER IF EXISTS trigger_schedule_trigger ON trigger_schedule;
+      
+      ALTER TABLE trigger_schedule
+          ALTER COLUMN TRIGGER TYPE public.trigger
+          USING TRIGGER::text::public.trigger;
+      
+      CREATE TRIGGER trigger_schedule_trigger
+          AFTER INSERT OR UPDATE OF trigger ON public.trigger_schedule
+          FOR EACH ROW
+          WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
+          EXECUTE FUNCTION public.add_event_to_trigger_queue ();
+      
+      DROP TRIGGER IF EXISTS review_assignment_trigger ON review_assignment;
+      
+      ALTER TABLE review_assignment
+          ALTER COLUMN TRIGGER TYPE public.trigger
+          USING TRIGGER::text::public.trigger;
+      
+      CREATE TRIGGER review_assignment_trigger
+          AFTER INSERT OR UPDATE OF trigger ON public.review_assignment
+          FOR EACH ROW
+          WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
+          EXECUTE FUNCTION public.add_event_to_trigger_queue ();
+      
+      DROP TRIGGER IF EXISTS review_trigger ON review;
+      
+      ALTER TABLE review
+          ALTER COLUMN TRIGGER TYPE public.trigger
+          USING TRIGGER::text::public.trigger;
+      
+      CREATE TRIGGER review_trigger
+          AFTER INSERT OR UPDATE OF trigger ON public.review
+          FOR EACH ROW
+          WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
+          EXECUTE FUNCTION public.add_event_to_trigger_queue ();
+      
+      DROP TRIGGER IF EXISTS verification_trigger ON verification;
+      
+      ALTER TABLE verification
+          ALTER COLUMN TRIGGER TYPE public.trigger
+          USING TRIGGER::text::public.trigger;
+      
+      CREATE TRIGGER verification_trigger
+          AFTER INSERT OR UPDATE OF trigger ON public.verification
+          FOR EACH ROW
+          WHEN (NEW.trigger IS NOT NULL AND NEW.trigger <> 'PROCESSING' AND NEW.trigger <> 'ERROR')
+          EXECUTE FUNCTION public.add_event_to_trigger_queue ();
+      
+      DROP TYPE public.trigger_old;    
+    `)
```

Comment:
A lot of code just to remove a couple of enum values! 🙄

Reply:
Wow 😮

Reply:
The only comment I have is on `trigger_action` the ones for ON_REVIEW_SELF_ASSIGN should be removed instead of replaced. But I guess it won't be a big problem to have the ON_REVIEW_ASSIGN duplicated, as it will when replacing the action